app/models.py

```diff
@@ -118,6 +115,7 @@ def __init__(self, **kwargs):
     uid = db.Column(db.String(64))  # DICOM Series UID (0020,000E)
     description = db.Column(db.String(100))  # DICOM Series Description (0008,103E)
     series_datetime = db.Column(db.DateTime)  # DICOM Series Date and Series Time
+    # These 2 make no sense to store, report can be looked up, archival doesn't help
```

Comment:
I'm not sure what the origin of these two entries was; imagine entries created from the Hackathon. Anyway:
- can understand `has_report` being useful for quickly checking things, albeit appreciate not imperative from your comment. Remember, there is a desire to use the databases for multi-centre analysis, so it might be useful in the db to have a column you can parse to check for reports.
- as for `archived`... I can't remember what this is for. At a guess, perhaps once there is a lot of image data in the hazen-web-app, it might be useful to have a functionality which retains the reports but not the images. That might be one purpose of an archive feature.

I'm happy either way, if preferable to retain these for future, or remove if they are currently unused.

Reply:
I'll leave these for now, has no negative impact on the rest of the application and can optimise db tables later.